pyproject.toml

```diff
@@ -60,3 +61,19 @@ requires = [
     "versioneer-518"
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+norecursedirs= [
+    ".*", "*.egg*", "build", "dist", "conda-recipe",
+]
+xfail_strict = true
+addopts = [
+    "--disable-warnings",
+    "--strict",
+    "--durations=20",
+    "-ra -vv",
```

Comment:
This option set through the command line, same for disable-warnings

Reply:
Removed the arguments that are specified in `pyproject.toml` from the CLI specified in the workflow when invoking `pytest`.